code/modules/surgery/organs/organ_internal.dm

```diff
@@ -362,6 +363,37 @@
 	owner.SetEyeBlurry(0)
 	owner.SetEyeBlind(0)
 
+/obj/item/organ/internal/robotize()
+	if(!(status & ORGAN_ROBOT)) //Don't override the icons for the already-mechanical IPC organs.
+		var/icon/temp = icon('icons/obj/surgery.dmi') //Insensitive to specially-defined icon files for species like the Drask or whomever else. Everyone gets the same robotic heart.
+		if(slot == "heart" && ("[slot]-prosthetic-on" in temp.IconStates()) && ("[slot]-prosthetic-off" in temp.IconStates())) //Give the robotic heart its robotic heart icons if they exist.
+			var/obj/item/organ/internal/heart/H = src
+			H.icon_base = "[slot]-prosthetic"
+			H.dead_icon = "[slot]-prosthetic-off"
+			H.update_icon()
```

Comment:
You search for states in a specific icon file, but don't set the organ to use that icon. Is that done elsewhere, or should you be setting the icon in here?